org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/editor/messages.properties

```diff
@@ -18,9 +18,11 @@ MavenTargetLocationWizard_12=Artifacts are resolved against the workspace <a hre
 MavenTargetLocationWizard_14=Label (optional)
 MavenTargetLocationWizard_9=Missing OSGi-Manifest
 MavenTargetLocationWizard_10=Dependencies scope
```

Comment:
What do you think about "Include dependencies of scope"  instead?

Reply:
Maybe but the labels shouldn't be too long ...